stable/database/templates/_helpers.tpl

```diff
@@ -497,3 +497,32 @@ Renders the name of the Secret for this database
 {{- define "database.secretName" -}}
 {{ .Values.admin.domain }}-{{ .Values.database.name }}
 {{- end -}}
+
+{{/*
+Renders the storage groups option for nuosm script
+*/}}
+{{- define "database.storageGroup.args" -}}
+{{- if .Values.database.sm.storageGroup }}
+{{- if .Values.database.sm.storageGroup.enabled }}
+{{- with .Values.database.sm.storageGroup.name }}
+{{- $invalid := list "ALL" "UNPARTITIONED" }}
+{{- if has (upper .) $invalid }}
+{{- fail (printf "Invalid storage group name: %s" .) }}
+{{- end }}
+{{- end }}
+- "--storage-groups"
+- {{ .Values.database.sm.storageGroup.name | default .Release.Name | trim | quote }}
```

Comment:
I guess this is supposed to accept a particular storage group rather than comma-separated list.

Reply:
This is a good point! The product supports passing multiple SGs as space space-separated list but we would like to avoid this. We want to enforce one-to-one mapping between Helm release and NuoDB storage group which simplifies deployment and operations. I will add a check for not allowing spaces.